app/views/make-meeting.jade

```diff
@@ -22,9 +22,9 @@ div.meeting
         button.icon(ng-click="addAttendee()" type="submit" analytics-on="click" analytics-event="Add Attendee")
           i.icon.icon-plus.icon--primary.success
 
-    div.col-lg-6.attendee-list
+    div.col-lg-8.attendee-list
       div(ng-if="!meeting.attendees.length")
-        span Enter attendee emails above and we'll grab their availability
+        span Enter your attendee's emails and we'll fill the calendar below with their availability.
```

Comment:
Add emails above and we'll update the calendar below to reflect everyone's schedule